### ISSUE TITLE
Error when running the examples in example.py

### ISSUE THREAD

**Issue by AnzAzam** [akapur/pyiqfeed#56]
Hello, thanks for your work on this library. I have followed all the steps specified in the readme and I was able to successfully get the news data using the -n flag, I am however not able to retrieve any other data with any of the other flags, when I tried that I got this error : **_No data returned because Request: HTX,SPY,100,0,H_0000000000,100
, Error: Unauthorized user ID._**

This error to me suggests that my authorization failed but I highly doubt that because when I ran the example with fake credentials, the result was different from when I ran it with the real credentials, in which case the iqfeed window opens up pre-filled and what seems to be authentication takes place. 

Can you please share your thoughts on the above error.

Thanks.

**Comment by akapur** [comment]
Sounds like a permissions problem. I would speak to DTN.

**Comment by AnzAzam** [comment]
You're right, after speaking with them I was able to solve this problem. Fantastic library by the way, my main challenge now is with getting the live data, specifically the level 1 streaming data using `get_level_1_quotes_and_trades()` and the live bar interval data using `get_live_interval_bars()`. I know that these functions call the `watch()` method and unlike the `HistoryConn` class don't directly return any data. I would like to know how I can return the data from the live API's to my application. Thanks.

**Comment by akapur** [comment]
Read the source, Luke. You have examples.
